tests/config.c

```diff
@@ -25,6 +25,9 @@
 #define TEST_CONFIG_DEFAULT_FANOUT                8
 #define TEST_CONFIG_DEFAULT_MAX_BRANCHES_PER_NODE 24
 
+#define TEST_CONFIG_DEFAULT_NUM_BACKGROUND_THREADS          (0)
```

Comment:
I think this and the next one can come out.

I think in a previous check-in to this file, I've added `TEST_CONFIG_DEFAULT_NUM_NORMAL_BG_THREADS` and `TEST_CONFIG_DEFAULT_NUM_MEMTABLE_BG_THREADS`, but forgot to take these two out.